mithril-aggregator/Dockerfile.ci

```
@@ -3,24 +3,27 @@
 # on a debian-compatible x86-64 environment
 FROM debian:11-slim
 
-# Args
-ARG CARDANO_BIN_URL=https://github.com/input-output-hk/cardano-node/releases/download/8.1.2/cardano-node-8.1.2-linux.tar.gz
+# Create appuser
+RUN adduser --no-create-home --disabled-password appuser
+
+# Precreate workdir
+RUN mkdir -p /app/bin
 
 # Upgrade
-RUN apt-get update -y && apt-get install -y libssl-dev ca-certificates wget && rm -rf /var/lib/apt/lists/*
+RUN apt-get update -y && apt-get install -y libssl-dev ca-certificates wget && apt-get clean && rm -rf /var/lib/apt/lists/*
 
-# Create appuser
-RUN adduser --no-create-home --disabled-password appuser
+# Install cardano-cli
+ARG CARDANO_BIN_URL=https://github.com/input-output-hk/cardano-node/releases/download/8.1.2/cardano-node-8.1.2-linux.tar.gz
+RUN wget -nv -O cardano-bin.tar.gz $CARDANO_BIN_URL \
+    && tar xzf cardano-bin.tar.gz ./cardano-cli && mv cardano-cli /app/bin \
+    && /app/bin/cardano-cli --version \
+    && rm -f cardano-bin.tar.gz
 
 # Copy the executable
 COPY mithril-aggregator/mithril-aggregator /app/bin/mithril-aggregator
-COPY mithril-aggregator/config /app/config
 
-# Install cardano-cli
-RUN wget -nv -O cardano-bin.tar.gz $CARDANO_BIN_URL
-RUN tar xzf cardano-bin.tar.gz ./cardano-cli && mv cardano-cli /app/bin
-RUN /app/bin/cardano-cli --version
-RUN rm -f cardano-bin.tar.gz
+# Copy the config files
+COPY config /app/config
```

It looks like some [tests failed in the CI](https://github.com/input-output-hk/mithril/actions/runs/6550181649/job/17789702156#step:6:186) because the config directory is incorrect: 
```suggestion
COPY mithril-aggregator/config /app/config
```